seedlet/assets/sass/blocks/button/_style.scss

```diff
@@ -39,6 +39,10 @@ input[type="submit"],
 			&.has-focus {
 				color: var(--button--color-background-hover);
 			}
+
+			&.has-text-color {
+				color: inherit;
```

Comment:
I'm not sure this is the correct fix. 

The problem is that when a preset custom text color is selected, the class `has-foreground-color[class]` is overridden by the button outline styles: 

<img width="425" alt="Screen Shot 2021-01-25 at 12 27 24 PM" src="https://user-images.githubusercontent.com/5375500/105741794-b8d0c980-5f08-11eb-9dea-98e0766032a4.png">

If I'm understanding correctly, setting to inherit just sets the link to the wrapping div's default color — which in Seedlet's case also happens to be `var(--global--color-foreground)`.

Reply:
Good catch — I just pushed a fix that only applies the color rule if the block doesn't include a `has-text-color` class. Mind retesting?

Reply:
That makes sense and looks good in my testing, thanks!